101-deploy-dsc-pullserver-to-win-server/azuredeploy.json

```diff
@@ -0,0 +1,304 @@
+                "Standard_GRS",
+                "Standard_ZRS"
+            ],
+            "metadata": {
+                "description": "The storage type created to store the VMs disks"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "West Europe",
+            "allowedValues": [
+                "East US",
+                "West US",
+                "West Europe",
+                "East Asia",
+                "South East Asia"
+            ],
+            "metadata": {
+                "description": "The location for the vms and other artefacts to be created."
+            }
+        },
+        "AdminUserName": {
+            "type": "string",
+            "defaultValue": "domadm",
+            "metadata": {
+                "description": "The admin user name for the VM created."
+            }
+        },
+        "AdminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The admin password for the VM created."
+            }
+        },
+        "WindowsOSVersion": {
+            "type": "string",
+            "defaultValue": "2012-R2-Datacenter",
+            "allowedValues": [
+                "2008-R2-SP1",
+                "2012-Datacenter",
+                "2012-R2-Datacenter",
+                "Windows-Server-Technical-Preview"
+            ],
+            "metadata": {
+                "description": "The version of the Windows OS for the VM."
+            }
+        },
+        "ImagePublisher": {
+            "type": "string",
+            "defaultValue": "MicrosoftWindowsServer",
+            "metadata": {
+                "description": "The publisher of the vm."
+            }
+        },
+        "ImageOffer": {
+            "type": "string",
+            "defaultValue": "WindowsServer",
+            "metadata": {
+                "description": "The image used for the Windows vm."
+            }
+        },
+        "VmSize": {
+            "type": "string",
+            "defaultValue": "Standard_D2",
+            "allowedValues": [
+				"Standard_A0",
+				"Standard_A1",
+				"Standard_A2",
+				"Standard_A3",
+                "Standard_A4",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4"
+			],
+            "metadata": {
+                "description": "The machine type for the server vm."
+            }
+        },
+        "dscPullSrvName": {
+            "type": "string",
+            "defaultValue": "pctst03pull",
+            "metadata": {
+                "description": "The name of the Windows vm."
+            }
+        },
+        "dscPullIPDnsName": {
+            "type": "string",
+            "defaultValue": "pctst03pubipull",
+            "metadata": {
+                "description": "The name of the public ip address"
+            }
+        },
+        "deployDSCPullServerConfigurationFile": {
```

Comment:
Make this a variable as the location of the file won't change. Also change the URI to this repo after the tempalte is merged